Bottom: 65d3408c6a4e402f067d010a5455223ab5ef250f
Top:    3440063a743f26e7d9b536eb809623198d132a30
Author: Alexandre Oliva <aoliva@redhat.com>
Date:   2016-04-01 06:20:02 -0300

Refresh of assorted-features.patch

---

diff --git a/include/gcc-c-interface.h b/include/gcc-c-interface.h
index e0af944..bc64152 100644
--- a/include/gcc-c-interface.h
+++ b/include/gcc-c-interface.h
@@ -133,7 +133,7 @@ struct gcc_c_fe_vtable
      provides the declaration.
 
      DATUM is an arbitrary piece of data that is passed back verbatim
-     to the callbakcs in requests.  */
+     to the callbacks in requests.  */
 
   void (*set_callbacks) (struct gcc_c_context *self,
 			 gcc_c_oracle_function *binding_oracle,
diff --git a/include/gcc-cp-interface.h b/include/gcc-cp-interface.h
index f6b80d2..72649a1 100644
--- a/include/gcc-cp-interface.h
+++ b/include/gcc-cp-interface.h
@@ -325,7 +325,7 @@ struct gcc_cp_fe_vtable
      name.
 
      DATUM is an arbitrary piece of data that is passed back verbatim
-     to the callbakcs in requests.  */
+     to the callbacks in requests.  */
 
   void (*set_callbacks) (struct gcc_cp_context *self,
 			 gcc_cp_oracle_function *binding_oracle,
